### DISCUSSION TITLE
Robot length and width change?

### DISCUSSION THREAD

**Discussion by wud2nuq** [mjansen4857/pathplanner#198] [Q&A] [answered]
With the windows app, although there is provision to edit the robot size in the setting menu from the 0.75m x 1.0m defaults, I cannot get the change to stick, even after creating a new project. I assume that I am missing something, and ask "how can you change the robot size from the default"? Thanks!

**Comment by mjansen4857** [comment, marked as answer]
You need to hit enter after changing a value in any text box in the app. I could make the settings menu save on close when I get around to it but you will still need to hit enter for other text boxes.

**Reply by wud2nuq** [reply]
Thanks, sorry I missed something so obvious!
